packages/odyssey-react/src/components/Button/Button.test.tsx

```diff
@@ -39,6 +39,20 @@ describe("Button", () => {
     expect(getByRole(button)).toHaveAttribute("disabled");
   });
 
+  it("renders an icon", () => {
+    render(
+      <Button
+        icon={
+          <svg>
+            <title>test</title>
+          </svg>
+        }
```

Comment:
I know that today we only support an icon on the left (start) side, in the future we will certainly need to support an icon on the end side as well.

Should we rename this `iconStart`? Just a thought, nothing to do here -- but worth considering.